target/linux/realtek/dts-5.10/rtl8382_zyxel_gs1900-24-v1.dts

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "rtl8380_zyxel_gs1900.dtsi"
+
+/ {
+	compatible = "zyxel,gs1900-24-v1", "realtek,rtl838x-soc";
```

Comment:
> Starting to look good to me. It might be good to have an RTL8382 DTSI at some point, so the ports don't have to be redefined every single time.

I may do this, actually. I suppose it would need to be part of a different PR?

I pulled the device tree almost entirely from the ZyXEL GS1900-24HP v2 and it worked immediately; I'm also preparing the ZyxEL GS1900-24HP v1 port. The device trees are almost all exactly identical bar the obvious (deleting the `poe` node for this board, changing the `memory`, etc.).

Reply:
... here's the tree for the GS1900-24HP v1, in case you're curious: https://github.com/hurricos/openwrt/tree/add_gs1900-24hp-v1